internal/function/is_tag.go

```diff
@@ -18,11 +18,9 @@ func NewIsTag(e sql.Expression) sql.Expression {
 	return &IsTag{expression.UnaryExpression{Child: e}}
 }
 
-var _ sql.Expression = (*IsTag)(nil)
```

Comment:
Why now isn't needed to check that it implements the `sql.Expression` interface?